lib/tasks/data.rake

```diff
@@ -0,0 +1,35 @@
+namespace :data do
+  desc 'Run all migrations'
+  task migrations: :environment do
+    Rake.application.in_namespace(:data) do |namespace|
+      namespace.tasks.each do |task|
+        next if task.name == 'data:migrations'
+
+        puts "Invoking #{task.name}"
+        task.invoke
+      end
+    end
+  end
+
+  desc 'Migrate user answers to responses'
+  task migrate_user_answers_to_responses: :environment do
+    puts 'Truncate responses'
+    ActiveRecord::Base.connection.execute('TRUNCATE responses RESTART IDENTITY CASCADE')
+    puts 'Migrating user answers to responses'
+    UserAnswer.all.each do |user_answer|
+      response = Response.create!(
+        user_id: user_answer.user_id,
+        training_module: user_answer.module,
+        question_name: user_answer.name,
+        answers: user_answer.answer,
+        correct: user_answer.correct,
+        assessments_type: user_answer.assessments_type,
```

Comment:
Storing type is not needed if we query because `training_module` and `question_name` will yield the correct Question. If we feel like storing the type is beneficial I believe we should take the opportunity to remedy the field `assessments_type`, because what we mean is `Question#page_type`. In the same vein I had anticipated this migtration might be accompanied by a change to assessments where we also see `assessments_type`. There it is redundant because all assessments are summative using summative questions.

Reply:
happy to discuss - this would intended originally to be a straight migration from one table to the other and I didn't look at user assessments as that seems unrelated to this migration.   How would you like to see assessments changed?